crypto/hd/algorithm_test.go

```diff
@@ -34,6 +35,7 @@ func init() {
 const mnemonic = "picnic rent average infant boat squirrel federal assault mercy purity very motor fossil wheel verify upset box fresh horse vivid copy predict square regret"
 
 func TestKeyring(t *testing.T) {
+	os.Setenv("GO_ETHEREUM_HDWALLET_FIX_ISSUE_179", "true")
```

Comment:
shouldn't this be 172?

Reply:
yup, but since the env have to match wallet one, free to try [!](https://github.com/miguelmota/go-ethereum-hdwallet/blob/7900505fafd0b747ce7835593373546f516df2e2/hdwallet_test.go#L17)